llvm/test/lab2/volodin_evgeniy/test.ll

```diff
@@ -0,0 +1,76 @@
+; RUN: opt -load-pass-plugin=%llvmshlibdir/InstrFuncVolodinE%shlibext -passes=instr-func-volodin -S %s | FileCheck %s
```

Comment:
Also check, that instrument_* funcs declarations are added.
And check a case when a function has already instrument_* calls